assignment2/milestone1/src/main/java/org/harsh/arya/easyimage/utils/job/JobHandler.java

```
@@ -0,0 +1,46 @@
+package org.harsh.arya.easyimage.utils.job;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.File;
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+
+public class JobHandler {
+
+    ObjectMapper mapper;
+    JobScheduler jobScheduler;
+
+    public JobHandler(){
+        mapper  = new ObjectMapper();
+        jobScheduler = new JobScheduler();
+    }
+
+    public Job[] retrieveJob(String filepath){
+        Job[] imageJob=null;
+        try{
+            imageJob = mapper.readValue(new File(filepath),Job[].class);
+        }
+        catch (JsonParseException e){
```

Its not good to catch and suppress these exceptions here, raise them

---

resolved